.storybook/themes/commerce.scss

```
@@ -880,9 +888,9 @@
 					grid-template-columns: initial,
 					grid-template-rows: 1fr,
 					gap: var(--global-spacing-5),
-					margin: auto,
-					max-width: calc(var(--content-max-width) * 1px),
-					width: var(--content-scale-width),
+					margin: 0,
+					max-width: var(--content-max-width),
+					width: var(--content-max-width),
```

You can't set these to the `--content-max-width` because that is a unitless value.  It should actually be stretching to fill the screen in mobile with no `max-width` which is the default characteristic.  Suggest editing these:
```suggestion
					max-width: 100%,
					width: 100%,
```
The desktop breakpoint style for this is setting `max-width` and `width` already so no change is needed there.

---

Sorry for the edit.  Due to a bug/implementation of the advanced styling, to make the desktop breakpoint work the values should have an initial value in the mobile breakpoint so I set these to 100%.

---

@nschubach Fixed it in [ce68305](https://github.com/WPMedia/arc-themes-blocks/pull/1515/commits/ce6830542c3e64bd463f5cb10e4ad5d8b45e666e)